packages/neuron-wallet/src/services/sync/get-blocks.ts

```diff
@@ -46,7 +46,12 @@ export default class GetBlocks {
         const checkTx = new CheckTx(tx, this.url)
         const addresses = await checkTx.check(lockHashes)
         if (addresses.length > 0) {
-          for (const input of tx.inputs!) {
+          const inputs = tx.inputs!
+          for (let i = 0; i < inputs.length; ++i) {
+            if (i === 0) {
+              continue
```

Comment:
For what reason the first input should be skipped?

Reply:
It's a cellbase and has no previous output.